### ISSUE TITLE
Enhancement - Glowing things should give resistance to SMV rather then cure / reduce it

### ISSUE THREAD

**Issue by pinkamedic** [loiste-interactive/Obenseuer-Issues#2247]
there are so many ways to cure fungal disease with the glowing things being the easiest way to do so, so they should give a temporay resistance to it for awhile rather then outright curing it

**Comment by Taco-Cat1** [comment]
I did some testing after seeing this and did some math. If the player were to go to every single glowing thing within 24 ingame hours which is possible it would cure their fungal disease completely even if they have less than a day to live. Note that I included the ones in Tenament A

**Comment by pinkamedic** [comment]
that more or less proves just how strong glowing things are, why bother going for any of the other options when the best and safest can be accessed with just some batteries (not to mention glowing things respawn on the daily)

**Comment by Nextej** [comment]
Some additional feedback from my and pinkamedic's discussion on discord. I'm too lazy to paraphrase
![image](https://github.com/loiste-interactive/Obenseuer-Issues/assets/44178400/481836d3-5fe9-429b-8ed2-d79e742ef29a)

**Comment by Taco-Cat1** [comment]
What if the more glowing things the player collects the less they heal if we don’t want to get resistance, the value would reset after 5-7 days

**Comment by Nextej** [comment]
> What if the more glowing things the player collects the less they heal if we don’t want to get resistance, the value would reset after 5-7 days

The issue here is that they would still heal you, which defeats purpose of other SMV treatments, if you collect a lot of things at once, even if every other will heal less, that's still gonna be a major SMV reduction, and if it would take only 5-7 days for the value to reset, player could still effectively be immune to SMV (not die of it) by only collecting red things regardless, so the original issue persists.

**Comment by Taco-Cat1** [comment]
Fair enough what if we went with the resistance but each one gives less resistance for less time
